src/encoded/static/components/search.js

```diff
@@ -1050,10 +1041,12 @@ export class FacetList extends React.Component {
         // are the negation facet terms that need to get merged into the regular facets that their
         // non-negated versions inhabit.
         const negationFilters = filters.filter(filter => filter.field.charAt(filter.field.length - 1) === '!');
-        
-        console.log("building facets");
-        
-        console.log(width);
+
+        // If there are no assay terms, the annotation type facet will need the "Method" label
+        // First we will check to see if there is an "assay_term_name" facet
+        const assayFacet = facets.find(facet => facet.field === 'assay_term_name');
+        // Then we will check to make sure that there is at least one "assay_term_name" term with "doc_count" greater than 0
+        const assayTerms = assayFacet ? assayFacet.terms.some(f => f.doc_count > 0) : false;
```

Comment:
Not that it makes a huge/any difference in performance, but you can shorten this to:

```
const assayTerms = assayFacet && assayFacet.terms.some(f => f.doc_count > 0);
```

If assayFacet is undefined, with this method you won’t get “false” but you will get falsy.